ohhell.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import List
+
+
+
+DECK_SIZE = 54
+with open("VERSION") as file:
+    GAME_VERSION = file.read()
+SCORE_MULTIPLIER = 10
+
+
+
+@dataclass
+class Player:
+    name: str
+    score: int
+    bids: int
```

Comment:
I think one thing here that makes the `Player` class a bit tricky here is the difference in how the `score` works compared to `bids` and `wins` here. `score` is a cumulative figure, and it's updated every round based on `bids` and `wins`, but `bids` and `wins` are dynamic, one-off figures. So you have one field that is deterministic based on the history of two other fields, but we don't store the history of those two other fields. Also since `score` is directly calculated from `bids` and `wins`, I don't think you can have it exist as another field like this. There's risk of them getting out of sync if that makes sense.

I think there are two ways to get around this
1. We have `bids` and `wins` both be `list[int]` rather than just `int`, and we remove `score` from the object and instead make either a `get_score()` method or a `score` property. 
2. We have a some object to capture the state of a single "round." That is if we want to store round level information like we are doing with `bids` and `wins`, I think this would not be a field on a `Player`, but rather some property of a `Round`. Not sure if that makes as much sense though.